distributed/parallelize_llama.py

```diff
@@ -62,7 +62,7 @@ def apply_tp(
     # after we apply TP to the model. Because we don't want to change model code 
     # when applying TP. We need to have change to ensure KVCache has the correct
     # size as k and v.
-    model.config.transformer_args["text"].n_local_heads = model.config.transformer_args["text"].n_local_heads // tp_mesh.size()
+    model.model.config.n_local_heads = model.model.config.n_local_heads // tp_mesh.size()
```

Comment:
`model.model` is really hard to reason about... 

The former was clunky, but model.model is worse

Reply:
"text" is not sustainable either, especially if the number of modules increases. It needs fixing, but this might not be perfectly there yet

Reply:
That's annoying, i'm 100% agree.
I will remove `model.model` as soon as I can.